tests/instruments/test_all_instruments.py

```diff
@@ -49,6 +49,17 @@
             if b:
                 devices.append(d)
 
+
+# Collect all properties
+# TODO add Channel properties as well: How collect all Channel classes?
```

Comment:
I guess that will be tricky, considering the generality of our approach towards channel collections.

Reply:
We could collect all `Channel` classes (and in that circumstance all `Instrument` classes) in the files below `pymeasure`.

Reply:
Shall we leave the collection of all the Channels for another PR?
I suggest so, because it was not that easy, when I first tried it with the instruments themselves.